registration/registration-services/src/main/java/io/mosip/registration/dto/RegistrationDTO.java

```diff
@@ -245,7 +247,15 @@ public void removeExceptionPhoto(String fieldId) {
 		    	iterator2.remove();
 		    }
 		}
-		
+
+		Iterator<Entry<String, Double>> iterator3 = this.SDK_SCORES.entrySet().iterator();
```

Comment:
let's rename 
"iterator" to bioCapturesIterator
"iterator2" to bioScoresIterator
"iterator3" to sdkScoresIterator